### PR TITLE
Alphabetically order commands in help files and add crafted examples

### PR DESCRIPTION
**Pull request by glGarg**
- Reorder commands in help files to make future diffing easier.
- Add new crafted examples to commands and groups without existing examples.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by marstr** [comment]
CI may be blocked by #8414 at the moment. Once it is merged, try rebasing on-top of upstream/dev and trying again.

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py [hunk 1]
```diff
@@ -52,11 +52,19 @@
 helps['appservice plan show'] = """
 type: command
 short-summary: Get the app service plans for a resource group or a set of resource groups.
+examples:
+  - name: Get the app service plans for a resource group or a set of resource groups. (commonly used with --output)
+    text: 'az appservice plan show --name MyAppServicePlan --resource-group MyResourceGroup  '
```
Comment (tjprescott) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py]: This shouldn't have the single quotes.

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py [hunk 2]
```diff
@@ -52,11 +52,19 @@
...
+    crafted: true
 """
 
 helps['appservice plan update'] = """
 type: command
 short-summary: Update an app service plan. See https://docs.microsoft.com/en-us/azure/app-service/app-service-plan-manage#move-an-app-to-another-app-service-plan to learn more
+examples:
+  - name: Update an app service plan. (crafted)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py]: (crafted) is unnecessary here. The customer won't have any idea what that means.

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py [hunk 3]
```diff
@@ -198,6 +222,10 @@
 helps['functionapp delete'] = """
 type: command
 short-summary: Delete a function app.
+examples:
+  - name: Delete a function app. (crafted)
+    text: az functionapp delete --name My --resource-group MyResourceGroup
```
Comment (tjprescott) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py]: `--name My`?  Is that even long enough to be valid in Azure?

**Comment by tjprescott** [comment]
A version of this PR has already been merged. Closing.

**Pull request closed without merging**